TravelFunds/2019.md

```diff
@@ -12,7 +12,7 @@ Benjamin | Gruenbaum | Collaborator Summit | Attendance | | Berlin | 30 May - 31
 Tobias | Nießen | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 500 € | 9 April 2019 | https://github.com/nodejs/admin/pull/333 ||$558.33|27 Jun 2019|12 Jul 2019|$558.33
 Dhruv | Jain | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 2000 USD | 5 April 2019 | https://github.com/nodejs/admin/pull/331 ||$1,817.03|26 Jun 2019|12 Jul 2019|$1,817.03
 Sagir | Khan | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 1600 USD | 4 April 2019 | https://github.com/nodejs/admin/pull/329 |||||
-Manuel | Spigolon | Collaborator Summit | Attendance | N/A | Berlin | 29 May - 02 Jun 2019 | 250€ | 20 April 2019 | https://github.com/nodejs/admin/pull/347 ||$246|26 Jun 2019|12 Jul 2019|$250
+Manuel | Spigolon | Collaborator Summit | Attendance | [🧾](https://github.com/nodejs/package-maintenance/issues/189#issuecomment-498441587) | Berlin | 29 May - 02 Jun 2019 | 250€ | 20 April 2019 | https://github.com/nodejs/admin/pull/347 |08 Jun 2019|$246|26 Jun 2019|12 Jul 2019|$250
```

Comment:
Looks good,
Only thing I'd say is to change document symbol "🧾" To "package-maintenance/issues/189"
So the link would become: [package-maintenance/issues/189](https://github.com/nodejs/package-maintenance/issues/189#issuecomment-498441587)

Then we can try to keep this format consistent for the whole document.

Reply:
Sure!